src/odemis/gui/cont/streams.py

```diff
@@ -2534,6 +2533,36 @@ def _updateMicroscopeStates(self):
                 self._tab_data_model.emState.value = guimodel.STATE_OFF
 
 
+class CryoStreamsController(StreamBarController):
+    """
+    Controls the display of stream panels without affecting the actual streams
+    """
+
+    def _onView(self, view):
+        """ Handle the changing of the focused view """
+        if not view or self.ignore_view:
+            return
+
+        # hide/show the stream panels which are compatible with the view
+        allowed_classes = view.stream_classes
+        for e in self._stream_bar.stream_panels:
+            ov_streams = self._tab_data_model.overviewStreams
+
```

Comment:
Please add a comment to explain what your `if` does. I think it tries to separate between different views, but which ones (there are 3 types)?